spec/unit/postmark/error_spec.rb

```diff
@@ -142,6 +149,36 @@
   it {is_expected.to be_a(Postmark::Error)}
 end
 
+describe(Postmark::InvalidEmailAddressError) do
+  describe '.new' do
+    let(:response) {{'Message' => message}}
+
+    subject do
+      Postmark::InvalidEmailAddressError.new(
+        Postmark::ApiInputError::INVALID_EMAIL_ADDRESS,
+        Postmark::Json.encode(response),
+        response)
+    end
+
+    let(:message) do
+      "{\"ErrorCode\":300,\"Message\":\"Error parsing 'To': Illegal email address " + \
+      "'ibalosh.testinggmail.com'. It must contain the '@' symbol.\"}"
+    end
+
```

Comment:
```suggestion
    let(:message) do
      "Error parsing 'To': Illegal email address 'ibalosh.testinggmail.com'. It must contain the '@' symbol."
    end

```

You want the message content here, not the original JSON response. 

The content of this string doesn't actually matter because you're not doing any assertions about it. I guess it's here just for documentation purposes.

Reply:
yeah, and oops, copy paste got in the way :)